src/pages/settings/Payments/AddDebitCardPage.js

```diff
@@ -301,7 +301,7 @@ class DebitCardPage extends Component {
                                     <StatePicker
                                         onChange={value => this.clearErrorAndSetValue('addressState', value)}
                                         value={this.state.addressState}
-                                        hasError={lodashGet(this.state.errors, 'addressState', false)}
+                                        errorText={this.getErrorText('addressState')}
```

Comment:
I just needed to use the Parent Component's getErrorText to pass `errorText` a `string`. This Picker is now working as expected.